### PR TITLE
adding usage with frontend tests of mysqli db for sitesearch credentials for wp sis plugin

### PR DESCRIPTION
**Pull request by bachka**

### CODE REVIEW

File: service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php [hunk 1]
```diff
@@ -2,22 +2,54 @@
...
+			// mockup our variables
+			$if_sis_siteId = "3bbf4db0-85ab-11e8-8c2f-3fec88e4efa0";
+			$if_sis_siteSecret = "4671e29a-85ab-11e8-9206-4b12904e274a";
+			// put the secrets to the wp_options table
+			add_option("intrafind-sis-siteId", $if_sis_siteId);
+			add_option("intrafind-sis-siteSecret", $if_sis_siteSecret);
+			// get our new options value from db table wp_options
+			$if_sis_siteId = get_option("intrafind-sis-siteId");
+			$if_sis_siteSecret = get_option("intrafind-sis-siteSecret");
+			echo "Alte Werte:".  $get_url_from_wp ."<br>". $if_sis_siteId ."<br>". $if_sis_siteSecret;
```
Comment (loxal) [inline service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php]: Let's use English only in our code :)

File: service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php [hunk 2]
```diff
@@ -2,22 +2,54 @@
...
+			update_option("intrafind-sis-siteId", $if_sis_siteId);
+			update_option("intrafind-sis-siteSecret", $if_sis_siteSecret);
+			$if_sis_siteId = get_option("intrafind-sis-siteId");
+			$if_sis_siteSecret = get_option("intrafind-sis-siteSecret");
+			echo "Neue Werte:".  $get_url_from_wp ."<br>". $if_sis_siteId ."<br>". $if_sis_siteSecret;
+		}
+	?>	
     <form method="post">
-        Website URL: <input type="text" id="sis-url" name="url" value="<?php echo $url; ?>">
+        Website URL: <input type="text" id="sis-url" name="url" value="<?php echo $get_url_from_wp; ?>">
```
Comment (loxal) [inline service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php]: Let's remove `name="url"` and `form` if we do not need it.

File: service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php [hunk 3]
```diff
@@ -2,22 +2,54 @@
...
+               onclick="registerSiteInSiS();">
+		<br><br>
+		<input type="submit" name="remove" value="Recycle DB Fields">
+	    <input type="submit" name="add" value="Add site credentials">
+		<input type="submit" name="update" value="Update site credentials">
     </form>
-    <?php
-    echo $url;
-    ?>
 </div><!-- wrapper div-->
```
Comment (loxal) [inline service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php]: Let's give this DIV and ID and call it sis-wrapper if you really want to give this DIV a name. This kind of comments are so 1990s ;)

File: service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php [hunk 4]
```diff
@@ -2,22 +2,54 @@
...
+			delete_option("intrafind-sis-siteSecret");	
+		}		
+		if(isset($_POST['add'])){
+			addSiS_Options_WP_DB();
+		}
+		if(isset($_POST['update'])){
+			updateSiS_Options_WP_DB();
+		}
+		function addSiS_Options_WP_DB(){	    		
+			// mockup our variables
```
Comment (loxal) [inline service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php]: Just `mock` :)

File: service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php (hunk 3 above)
Comment (loxal) [reply service/src/main/resources/static/external/wordpress-plugin/sitesearch/sis-admin-page.php]: `<!-- wrapper div-->`

**Comment by bachka** [comment]
now should be all the files in correct places ...

**Pull request merged**